### ISSUE TITLE
Several error messages - complex log

### ISSUE THREAD

**Issue by folrewef** [keesschollaart81/vscode-home-assistant#217]
Please include the related YAML code:

```YAML
your:
   - yaml
```

Find my Home Assistant Configuration here:
[Link to Home Assistant Configuration](<link to repo here>)

Extension runs in:
- [X] VS Code for Desktop 
- [ ] Hassio Add-on 
- [ ] Visual Studio Online
- [ ] Other: 

I'm running VS Code on:
- [X ] Windows 
- [] Mac 
- [ ] Linux 

I'm accessing my files:
- [ ] From local disk 
- [ ] Via remote SSH 
- [ X] Smb
- [ ] Other:

**Comment by folrewef** [comment]
I do not know why this happens.

I get the following 3 error messages upon start:

Error message 1:

> UnhandledPromiseRejectionWarning: Error: [UriError]: Scheme contains illegal characters. at _validateUri (c:\Users\Fredrik\.vscode\extensions\keesschollaart.vscode-home-assistant-1.5.0\node_modules\vscode-uri\lib\umd\index.js:68:19) at _URI.URI (c:\Users\Fredrik\.vscode\extensions\keesschollaart.vscode-home-assistant-1.5.0\node_modules\vscode-uri\lib\umd\index.js:157:17) at new _URI (c:\Users\Fredrik\.vscode\extensions\keesschollaart.vscode-home-assistant-1.5.0\node_modules\vscode-uri\lib\umd\index.js:359:51) at Function.URI.parse (c:\Users\Fredrik\.vscode\extensions\keesschollaart.vscode-home-assistant-1.5.0\node_modules\vscode-uri\lib\umd\index.js:269:20) at ConfigurationService.getUri (c:\Users\Fredrik\.vscode\extensions\keesschollaart.vscode-home-assistant-1.5.0\out\language-service\src\configuration.js:20:37) at ConfigurationService.updateConfiguration (c:\Users\Fredrik\.vscode\extensions\keesschollaart.vscode-home-assistant-1.5.0\out\language-...


Error message 2

> UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)

Error message 3:

> [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.

**Comment by folrewef** [comment]
Furthermore I got the following in the output window:

[Home Assistant Language Server(15828)] Started and initialize received
Applying schema's to 10 of your configuration files...
Schema's updated!
Discover all configuration files because file:///z%3A/automations.yaml got updated and new files were found...
Applying schema's to 10 of your configuration files...
Schema's updated!
Discover all configuration files because file:///z%3A/automations.yaml got updated and new files were found...
Connecting to Home Assistant...
[Auth phase] Initializing WebSocket connection to Home Assistant
[Auth Phase] Connecting to Home Assistant...
[Error - 14:00:16] Request textDocument/completion failed.
  Message: Request textDocument/completion failed with message: Invalid URL: wsfined/api/websocket
  Code: -32603 
Connecting to Home Assistant...
[Auth phase] Initializing WebSocket connection to Home Assistant
[Auth Phase] Connecting to Home Assistant...
[Error - 14:00:16] Request textDocument/completion failed.
  Message: Request textDocument/completion failed with message: Invalid URL: wsfined/api/websocket
  Code: -32603 
Applying schema's to 10 of your configuration files...
Schema's updated!
Discover all configuration files because file:///z%3A/automations.yaml got updated and new files were found...
Applying schema's to 10 of your configuration files...
Schema's updated!
